woocommerce/woocommerce
# fix target url of 'Manage Suggestions', settings are now in 'Advanced'

### All Submissions:


### Changes proposed in this Pull Request:
Fixes the URL for `Manage suggestions` link. The relevant settings are now in WooCommerce > Settings > Advanced.

The footer url was also hard-coded in the js; this may cause issues (e.g. on multisite install). This PR generates that url using `admin_url` and passes down to client as "localise" data.

### How to test the changes in this Pull Request:
1. View product edit page, scroll down to metabox. Click `Get more options` tab.
2. Click `Manage suggestions` in footer - should lead to an opt-out UI.
3. View product edit page, scroll down to metabox. Click `Get more options` tab.
2. Dismiss all suggestions, so 'no suggestions' state is displayed. (See second image)
1. Click `Manage suggestions` in no-suggestions state - should lead to an opt-out UI.

<img width="562" alt="Screen Shot 2019-04-10 at 12 06 05 PM" src="https://user-images.githubusercontent.com/4167300/55842772-0ab82c00-5b89-11e9-998a-e06f202fde52.png">

<img width="927" alt="Screen Shot 2019-04-10 at 12 05 35 PM" src="https://user-images.githubusercontent.com/4167300/55842750-f7a55c00-5b88-11e9-969b-f534c6475579.png">
